commons/data_access_layer/cosmos_db.py

```diff
@@ -251,19 +250,13 @@ def find_all(
         self,
         event_context: EventContext,
         conditions: dict = None,
-        custom_sql_conditions: List[str] = None,
-        custom_params: dict = None,
+        date_range: dict = None,
```

Comment:
same as above, date range.

Reply:
This method has its own ticket to attend its refactoring, please see https://ioetec.atlassian.net/browse/TT-245, nevertheless, I will take account of the suggestion.